### DISCUSSION TITLE
Add alt languages right next to English named languages and change Euskera into Basque (typing language)

### DISCUSSION THREAD

**Discussion by Whydidthechickencrosstheroad** [monkeytypegame/monkeytype#5683] [Ideas]
For people who do not speak English, it is a issue. When you are selecting a language to type, there are options, for example: Spanish

It should add a translation in that language.
Now, when added a mention of the translation, it should say Spanish / Español
Or it could be said a another way. Any code or programming languages will not count. Git or Twitch emotes, League of Legends, or typing of dead language option for typing will not count as the suggestion.

For Euskera, it should say Basque / Euskera because sometimes people might not know what language named Euskera is

**Comment by fehmer** [comment]
So you want every (non-english, non-code) language to have a translation of the language name in that language?

e.g.
|english|translated|
|-------|------|
| spanish | español |
| finnish |   suomi |
| turkish |  türkçe |
| swedish | svenska |
| german  | deutsch |
| polish  | polski  |

**Reply by Whydidthechickencrosstheroad** [reply]
Yes, like that. It also has to be right next to the English named language. Code or programming languages will not be included or Git, Twitch emotes, League of Legends, or typing of dead will actually not be included

**Reply by Whydidthechickencrosstheroad** [reply]
It has been 3 weeks and still not added
